src/Model/CommandResponse.php

```diff
@@ -0,0 +1,56 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\Model;
+
+use Lmc\Matej\Exception\InvalidDomainModelArgumentException;
+
+/**
+ * Response to one single command which was part of request batch.
+ */
+class CommandResponse
+{
+    const STATUS_OK = 'OK';
+    const STATUS_ERROR = 'ERROR';
+    const STATUS_SKIPPED = 'SKIPPED';
+
+    /** @var string */
+    private $status;
+    /** @var string */
+    private $message;
+    /** @var array */
+    private $data = [];
+
+    private function __construct()
```

Comment:
Prázdný konstruktor?

Reply:
A private empty constructor means that this class cannot be instantiated.